front/src/services/giphy.service.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const baseUrl = 'https://api.giphy.com/v1/gifs/search';
+const type = 'gifs';
+const apiKey = '3rgXBKNKDmcA5Ykg9i';
```

Comment:
sill constrained to have apiKey in the code ?
maybe we can put it in db and encode it ?

Reply:
For the moment is the giphy extension key. Can we do that in a near futur ?

Reply:
for sure !